src/components/CardListBox/CardFolderList/CardFolderList.tsx

```diff
@@ -0,0 +1,35 @@
+import Folder from "./Folder/Folder";
+import styles from "./CardFolderList.module.scss";
+import { ObjectFolder } from "@/constants/index.types";
+
+interface CardFolderListProps {
+  folders: ObjectFolder[];
+  onFolderClick: (folder: ObjectFolder) => void;
```

Comment:
변수명에 자료형을 포함시키는 것을 헝가리안 케이스라고 표현하는데요!
Object-라는 표현이 없어도 데이터 타입을 쉽게 추측할 수 있고, 오히려 Object-라는 의미가 추가되면서 누군가에게는 헷갈릴수도 있다는 생각이 들어서 아래와 같이 표현해줘도 괜찮을 것 같아요!
```suggestion
  onFolderClick: (folder: Folder) => void;
```

Reply:
수정을 진행하면 컴포넌트이름과 곂쳐 에러가 나는 경우인듯 한데 혹시 다른 직관적인 컨벤션이 있을까요?

Reply:
다른 코멘트에서 의견 남겼습니다!